### PR TITLE
Add a script to combine all latency result CSVs.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: test/e2e/ig_latency/ig_latency.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
+method,no-import-group,import-group,change-rate
+GetLocationsRankings,2.059969,3.296367,160.020245%
```
Comment (shifucun) [inline test/e2e/ig_latency/ig_latency.csv:2]: Some minor formatting nit, maybe keep 2 decimal for second, and show change rate as +60.1% or -20.1% (the actual change percent)

File: docs/developer_guide.md [hunk 2]
```diff
@@ -190,6 +190,12 @@ Or, run the following command for latency test for a specific API endpoint:
 go test -parallel 1 path/to/the/test/file.go
 ```
 
+A CSV file is generated for each API method. To combine all of them:
+
+```bash
+./test/e2e/combine_ig_latency.sh
```
[the remark this answers is not in the document]
Comment (Spaceenter) [reply docs/developer_guide.md]: done

File: test/e2e/ig_latency/ig_latency.csv (hunk 1 above)
Comment (Spaceenter) [reply test/e2e/ig_latency/ig_latency.csv]: done

**Review by shifucun — APPROVED** [review]
thanks for the update!

**Pull request merged**